blocks/reader-revenue-manager/components/SettingsForm.js

```diff
@@ -0,0 +1,115 @@
+/**
+ * Reader Revenue Manager Block Editor SettingsForm component.
+ *
+ * Site Kit by Google, Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+// eslint-disable-next-line import/no-unresolved
+import { SelectControl } from '@wordpress-core/components';
+// eslint-disable-next-line import/no-unresolved
+import { Fragment, useState } from '@wordpress-core/element';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { MODULES_READER_REVENUE_MANAGER } from '../../../assets/js/modules/reader-revenue-manager/datastore/constants';
+
+const { select, dispatch } = Data;
+const CORE_EDITOR = 'core/editor';
```

Comment:
Let's extract this constant to `blocks/reader-revenue-manager/common/constants.js`, in line with the changes that are coming in https://github.com/google/site-kit-wp/pull/10247/files#diff-eebc7bb1047fb0c0e7a220332932857840c7bfdba8549a67ab518a829497024f.